docs/field-details.asciidoc

```diff
@@ -8003,7 +8003,9 @@ Note also that the `user` fields may be used directly at the root of the events.
 [discrete]
 ==== User Field Usage
 
-For usage and examples of the user fields please see the <<ecs-user-usage,usage>> section.
+For usage and examples of the user fields please see the 
```

Comment:
Newlines introduced into the Jinja2 template file will be included in the generated output. The [final product](https://ecs_1354.docs-preview.app.elstc.co/guide/en/ecs/master/ecs-user.html#_user_field_usage) on the docs site looks ok, though. Was that done intentionally here?

Even if it's just the asciidoc files, I'd prefer the formatting not to contain unnatural breaks. I think it makes the docs more readable anytime we do have to look at their raw form and makes code diffs easier in reviews. On the other hand, it can, unfortunately, make the `j2` template files a little uglier.

Reply:
No worries, I've concatenated in the template & rebuilt